internal/component/loki/source/kubernetes/kubetail/target.go

```diff
@@ -25,6 +25,21 @@ const (
 	kubePodUID           = "__meta_kubernetes_pod_uid"
 )
 
+var ClusteringLabels = []string{
```

Comment:
Do I understand correctly that these are the labels which uniquely identify a log? Are we sure that we didn't miss any? What if the user also has some other custom labels?

Reply:
Technically its just name, namespace, container name for the tailer. But we use UID to identify the pod in the positions file so I wanted to add that to ensure it is sharded to the correct pod.